src/layouts/css-component-layout.tsx

```diff
@@ -0,0 +1,118 @@
+      }
+    }
+  }
+`
+
+export default function CssComponentLayout({data}) {
+  console.log(data.sitePage)
+  const name = data.sitePage?.context.frontmatter.cssId || ''
+  const title = data.sitePage?.context.frontmatter.title || name
+  const description = data.sitePage?.context.frontmatter.description || ''
+  const stories = [{id: `deprecated-${name}--default`}]
+
+  return (
+    <BaseLayout title={title} description={description}>
+      <Box sx={{maxWidth: 1200, width: '100%', p: [4, 5, 6, 7], mx: 'auto'}}>
+        <Heading as="h1" sx={{fontSize: 7}}>{title}</Heading>
+        {description ? (
+          <Text as="p" sx={{fontSize: 3, m: 0, mb: 3, maxWidth: '60ch'}}>
+            {description}
+          </Text>
+        ) : null}
+        <Box sx={{mb: 4}}>
+          <ComponentPageNav
+            basePath={data.sitePage.path}
+            includeReact={data.sitePage.context.frontmatter.reactId}
+            includeRails={data.sitePage.context.frontmatter.railsIds}
+            includeFigma={data.sitePage.context.frontmatter.figmaId}
+            includeCSS={data.sitePage.context.frontmatter.cssId}
+            current="css"
+          />
+        </Box>
+        <Note variant="warning">
+          <Text sx={{display: 'block', fontWeight: 'bold', mb: 2}}>Primer CSS is no longer actively maintained</Text>
+          <Text>The <Link as={GatsbyLink} to="https://github.com/primer/css">CSS library</Link> is still available, but these components will not receive new features or major changes moving forward. We encourage you to use Primer React or View Components wherever possible.</Text>
+        </Note>
+        <Box sx={{display: 'flex', mt: 4, alignItems: 'start', gap: [null, 7, 8, 9]}}>
+          <Box sx={{width: '100%'}}>
+            {/* @ts-ignore */}
+            <Box
+              sx={{
+                display: 'flex',
+                flexDirection: ['column', null, null, null, 'row'],
+                justifyContent: 'space-between',
+                gap: 3,
+                mb: 4,
+              }}
+            >
+              <Box
+                as={'ul'}
+                sx={{
+                  display: 'flex',
+                  gap: 3,
+                  alignItems: 'center',
+                  m: 0,
+                  p: 0,
+                  paddingInline: 0,
+                  listStyle: 'none',
+                  fontSize: 1,
+                  '& > li': {
+                    display: 'flex',
+                  },
+                }}
+              >
+                <SourceLink href={`https://github.com/primer/css/blob/main/src/${name}`} />
+                {stories.length > 0 ? (
+                  <StorybookLink href={`https://primer.style/css/storybook/?path=/story/${stories[0].id}`} />
+                ) : null}
+              </Box>
+            </Box>
+
+            <H2>Examples</H2>
+            {stories.length > 0 ? (
+              <StorybookEmbed framework="css" height={300} stories={stories} />
```

Comment:
can't seem to get the `Show Code` button to appear on the embed...is that bc of CSS Storybook configuration?

Reply:
It's not Storybook related. It seems the `code` data is passed and generated on the React side and it's part of the component.json file: https://unpkg.com/@primer/react@35.27.1/generated/components.json

React code docs generation: https://github.com/primer/react/blob/main/script/components-json/build.ts#L119

If we want the `Show Code` we'll need to generate a similar components.json file including the code used in the stories in the CSS repo 😄

Reply:
This seems like a good chunk or work - let's handle in a follow-up PR?

Reply:
I think it's ok to do this in a follow-up PR